deploy/operator/mirror_utils.sh

```diff
@@ -66,6 +66,19 @@ EOF
   cat "${manifests_dir}/catalogSource.yaml"
 
   oc apply -f "${manifests_dir}/catalogSource.yaml"
+
+  # Until allowing mirroring by tags https://issues.redhat.com/browse/OCPNODE-521
+  # https://github.com/openshift/api/pull/874 will be part of OCP 4.9
+  hostnames=$(virsh net-dhcp-leases ${BAREMETAL_NETWORK_NAME} | awk 'NR>2 {print $6}')
+
+  for hostname in ${hostnames}; do
+      ssh -o StrictHostKeyChecking=no core@${hostname} bash - << EOF
+  if grep "mirror-by-digest-only = true" /etc/containers/registries.conf -c; then
+      sudo sed -i 's/mirror-by-digest-only = true/mirror-by-digest-only = false/' /etc/containers/registries.conf
+      sudo systemctl restart crio kubelet
+  fi
```

Comment:
you don't need/want that on CI

Reply:
What's the harm?

Reply:
having some of those things break (dev-scripts dhcp leases change, registries conf changing location, etc.) and us being broken on it
we should strive to use only official API on CI and locally. Report the digest as bug and it will be handled on community operators